### PR TITLE
Add Manual Sentinel Incident Workbook

### PR DESCRIPTION
**Pull request by pthoor**
Required items, please complete
   
   Change(s):
   - Added new workbook (Manual Sentinel Incident via ARM Action)

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Looking into this

**Review by v-prasadboke — DISMISSED** [review]

**Review by v-prasadboke — COMMENTED** [review]
Please resolve the validation checks

File: Workbooks/ManualSentinelIncident.json [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+            "type": 11,
+            "content": {
+              "version": "LinkItem/1.0",
+              "style": "list",
+              "links": [
+                {
+                  "id": "0ce7d423-77f7-4221-b79b-787164bb4935",
+                  "linkTarget": "ArmAction",
+                  "linkLabel": "Create Incident",
+                  "preText": "❗",
```
Comment (v-prasadboke) [inline Workbooks/ManualSentinelIncident.json:218]: Hello @pthoor The validation check might be failing due to "pretext": "!"
Is this string valid or should be changed

**Comment by v-prasadboke** [comment]
Hello @pthoor any updates for the validation errors

**Comment by v-prasadboke** [comment]
Hello @pthoor any solution to resolve the validation checks

**Comment by pthoor** [comment]
> Hello @pthoor any solution to resolve the validation checks

Will try to have another version without emojis to see if that's the case why the checks are failing.

**Comment by v-prasadboke** [comment]
Sorry @pthoor for the late reply 
Checking on this now

**Comment by v-prasadboke** [comment]
Hello @pthoor can you please the metadata for this workbook into workbookmetadata file https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json

**Review by v-prasadboke — APPROVED** [review]

**Comment by v-prasadboke** [comment]
Hello @pthoor all looks good

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**